src/sentry/features/temporary.py

```diff
@@ -158,6 +158,7 @@ def register_temporary_features(manager: FeatureManager):
     manager.add("organizations:issue-stream-empty-state-additional-platforms", OrganizationFeature, FeatureHandlerStrategy.FLAGPOLE)
     # Enable issue stream performance improvements
     manager.add("organizations:issue-stream-performance", OrganizationFeature, FeatureHandlerStrategy.REMOTE)
+    manager.add("organizations:issue-search-snuba", OrganizationFeature, FeatureHandlerStrategy.REMOTE)
```

Comment:
just curious, why is `REMOTE` used instead of `FLAGPOLE`?

Reply:
oops old habits